plenary/plenary-21.yaml

```diff
@@ -0,0 +1,287 @@
+    # - BE Mr. J.-L. Champion
+    # - CH Mr. F. Vuilleumier
+    # - DE Mr. M. Dill
+    # - GB Ms. S.L. Probert
+    # - JP Mr. K. Wakaizumi
+    # - KR Mr. Y.-G. Jung
+    # - US Ms. M.K. Blantz
+    # in co-operation with the following ISO/TC154 L(iaison)-members:
+    # - UPU (standard S42),
+    # - CEN/TC331 (standard CEN14142)
+    # - ISO/TC215 as well as other interested TC’s
+    # - ISO/IEC JTC1/SC32 as well as other interested SC’s
+    # - UN/CEFACT Forum: at least TBG, possibly ATG
+    # - UNECE UNeDocs Project
+    # - OASIS Management and UBL TC + CIQ TC
+    # - OAGI
+
+
+  - dates:
+      - 2003-09-12
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "ISO TS16668:2000 Basic Semantics Register BSR"
+    identifier: 245
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: Approved
+
+    actions:
+      - type: notes
+        date_effective: 2003-09-12
+        message: notes that the ISO TS16668:2000 has not been widely implemented by ISO members
+      - type: notes
+        date_effective: 2003-09-12
+        message: notes that this TS is also largely superseded by the latest deliverables of ISO/IEC JTC1/SC32 on ISO11179 (and related products), as well as current developments of ebXML parts, especially by OASIS
+      - type: resolves
+        date_effective: 2003-09-12
+        message: resolves to withdraw ISO TS11168:2000
+
+
+  - dates:
+      - 2003-09-12
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "ISO NP22208:2000 Rules for the operation of EDI/EC Registration Authorities"
+    identifier: 246
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: Approved
+
+    actions:
+      - type: recalls
+        date_effective: 2003-09-12
+        message: recall that this New Project ISO NP22208 (EDIRA) has been launched in 2000-07
+      - type: notes
+        date_effective: 2003-09-12
+        message: notes that EDIRA has since abandoned this ISO project
+      - type: notes
+        date_effective: 2003-09-12
+        message: notes that, in 2003-07, NP22208 has been withdrawn from the work program of ISO/TC154
+
+
+  - dates:
+      - 2003-09-12
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    identifier: 247
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: Approved
+
+    actions:
+      - type: regrets
+        date_effective: 2003-09-12
+        message: ISO/TC154 deeply regrets the passing of friend and colleague Michael Adcock. Mike added calm, logic, humor, and wisdom to the group’s deliberations and his presence will always be sorely missed by the members of our Committee
```

Comment:
In this part, I used the word `regrets` as action type. Is that correct?

Reply:
Agreed.